### DISCUSSION TITLE
1.1 - Removed Flipping

### DISCUSSION THREAD

**Discussion by earthtraveller1** [earthtraveller1/tetris-cli#8] [Announcements]
Okay, so apparently, you are not allowed to flip pieces in tetris, only rotate them by 180 degrees. So, yeah, this release basically gets rid of those and replaces them with 180 degree rotations.

## Changes

- Removed the ability to flip pieces.
- Added the ability to rotate pieces by 180 degrees.
- Updated the controls list on the right.

<hr /><em>This discussion was created from the release <a href='https://github.com/earthtraveller1/tetris-cli/releases/tag/1.1'>1.1 - Removed Flipping</a>.</em>

**Comment by nli33** [comment]
:moai:

**Reply by earthtraveller1** [reply]
Indeed.
